src/main/scala/com/databricks/spark/avro/AvroRelation.scala

```diff
@@ -15,21 +15,27 @@
  */
 package com.databricks.spark.avro
 
+import java.util.Map
+import java.nio.ByteBuffer
+
 import org.apache.avro.Schema
 import org.apache.avro.file.DataFileReader
 import org.apache.avro.generic.{GenericRecord, GenericDatumReader}
 import org.apache.avro.mapred.FsInput
+import org.apache.avro.util.Utf8
+import org.apache.avro.generic.GenericData
+import org.apache.avro.generic.GenericData.{Fixed, EnumSymbol, Record}
 
 import org.apache.hadoop.fs.{FileSystem, Path}
 
 import org.apache.spark.sql._
-import org.apache.spark.sql.sources.{TableScan, BaseRelation}
+import org.apache.spark.sql.sources.TableScan
 
 import scala.collection.JavaConversions._
 
 case class AvroRelation(location: String)(@transient val sqlContext: SQLContext) extends TableScan {
 
-  val schema = {
+  override val schema = {
     val fileReader = newReader()
     val convertedSchema = toSqlType(fileReader.getSchema).dataType match {
```

Comment:
We should probably note in the documentation that the code currently assumes the schema is the same for all of the avro files in a directory.